_drafts/_heat/Heat_Orchestration_Template_LB.yaml

```diff
@@ -0,0 +1,109 @@
+heat_template_version: 2013-05-23
+description: Simpler server with lb
+parameters:
+  image:
+    type: string
+    description: Image used for servers
+    default: Red Hat Enterprise Linux 6.5 x86_64
+  flavor:
+    type: string
+    description: flavor used by the web servers
+    default: Standard_1_2
+  network_subnet_lb_pool:
+    type: string
+    description: Subnet to configure load balancer
+    default: 8b3c5b67-a04e-4159-b805-753bc2c56912
+  network_server:
+    type: string
+    description: Network to which the server will be attached
+    default: 8d601329-4a38-4a85-bc84-ba7d2725ebd9
+  az:
+    type: string
+    description: availability zone for instances
+    default: nova
+
+
+resources:
+
+  my_server:
+    type: OS::Nova::Server
+    properties:
+      flavor: {get_param: flavor}
+      image: {get_param: image}
+      availability_zone: {get_param: az}
+      networks:
+        - network: { get_param: network_server }
+      user_data_format: RAW
+      user_data: |
+        #!/bin/sh
+        #yum install -y httpd
+        service httpd start
+        echo "IBM Cloud Openstack Services Demo: " > /var/www/html/index.html        
```

Comment:
The product name is not IBM Cloud Openstack Services any more. This should be changed accordingly.